### PR TITLE
fixing link to create a pull request

### PR DESCRIPTION
**Pull request by All4Gis**

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -730,7 +730,7 @@ successful install, they're not needed anymore.
 In order to check out QGIS sources from the repository, you need a git client.
 This installer should work fine:
 
-http://msysgit.googlecode.com/files/Git-1.7.4-preview20110204.exe
+https://github.com/git-for-windows/git/releases/download/v2.13.2.windows.1/Git-2.13.2-32-bit.exe (32bit) or https://github.com/git-for-windows/git/releases/download/v2.13.2.windows.1/Git-2.13.2-64-bit.exe (64bit)
```
Comment (DelazJ) [inline INSTALL]: How about sending instead to that page https://git-scm.com/downloads and let user pick the version that will be available at that time? There's imho more chance to have it uptodate (with latest release).

**Comment by m-kuhn** [comment]
I think this file is generated from the .t2t files in the doc folder

File: INSTALL (hunk 1 above)
Comment (All4Gis) [inline INSTALL]: Okey,this is a good option,but you should then change more links pointing to a particular version,for example cmake link

**Comment by All4Gis** [comment]
Oops!!Yes the correct file is : https://github.com/qgis/QGIS/blob/master/doc/msys.t2t#L118.
If you want close this PR and I wil create other ,change this line using the @DelazJ comment.

**Comment by m-kuhn** [comment]
You can just push to your own master branch and this pull request here will be updated.

**Comment by m-kuhn** [comment]
Thanks

**Pull request merged**